docs/resources/security_monitoring_rule.md

```diff
@@ -1,16 +1,17 @@
 ---
-page_title: "datadog_security_monitoring_rule"
+page_title: "datadog_security_monitoring_rule Resource - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  Provides a Datadog Security Monitoring Rule API resource. This can be used to create and manage Datadog security monitoring rules. To change settings for a default rule use datadogsecuritydefault_rule instead.
 ---
 
-# datadog_security_monitoring_rule Resource
+# Resource `datadog_security_monitoring_rule`
 
-Provides a Datadog [Security Monitoring Rule API](https://docs.datadoghq.com/api/v2/security-monitoring/) resource. This can be used to create and manage Datadog security monitoring rules. To change settings for a default rule use [datadog_security_default_rule](/resources/security_monitoring_default_rule) instead.
+Provides a Datadog Security Monitoring Rule API resource. This can be used to create and manage Datadog security monitoring rules. To change settings for a default rule use datadog_security_default_rule instead.
```

Comment:
```suggestion
Provides a Datadog Security Monitoring Rule API resource. This can be used to create and manage Datadog security monitoring rules. To change settings for a default rule use `datadog_security_default_rule` instead.
```